__tests__/app.test.js

```diff
@@ -261,4 +261,39 @@ describe("GET: /api/reviews", () => {
             expect(reviews).toBeSorted({ key: 'created_at', descending: true})
         });
     });
+});
+
+//A test battery thats ensures the review_id/comments endpoint returns the correct propeties.
+describe("GET: /api/reviews/:reviews_id/comments", () => {
+    test("Status 200: responds with an array of objects containing the appropriate properties", () => {
+        return request(app)
+        .get("/api/reviews/3/comments")
+        .expect(200)
+        .then(({ body }) => {
+            const {comment} = body;
+            expect(comment).toBeInstanceOf(Array);
+            expect(comment).toHaveLength(3);
+                comment.forEach((comments) => {
+                expect(comments).toEqual(expect.objectContaining(
+                    {
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    review_id: expect.any(Number),
+                    }
+                ))
+            });
+        });
+    });
+    test.only("Status 200: found a review but no comments to show", () => {
+        return request(app)
+        .get("/api/reviews/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.comment).toEqual("msg: no comments on this review")
+
+        })
+    })
```

Comment:
Being rather picky but I would probably expect an empty array of comments rather than this message. Especially if you are going to send back on a key of comment. Due to this naming convention I would expect there to be data relating to comments.

I understand that you might want to communicate to a customer when there are no comments but I would say that's a job for the front end

Reply:
Also ... don't forget about your errors tests and code 😁

Reply:
OK will change  it to an empty array :)

I have just realised I didn't send my latest commit! I've already done the tests and code! Have pushed it now :)